spec/src/main/asciidoc/repository.asciidoc

```diff
@@ -786,19 +791,19 @@ In this scenario, we have the following data models:
 class Order {
   private Long id;
   private String addressZipCode;
-  private Address address;
+  private MailingAddress address;
 }
 
-class Address {
-  private Zipcode zipcode;
+class MailingAddress {
+  private int zipcode;
 }
 ----
 
-- The query method `findByAddress_ZipCode` takes a `Zipcode` object as a parameter.
+- The query method `findByAddress_ZipCode` takes an `int` object as a parameter.
 - The method name includes an underscore (`_`) indicating manual traversal points.
 - The Property Resolution Algorithm will parse the method name and extract `Address_ZipCode`, recognizing the underscore as a traversal point.
-- It will then attempt to resolve the property named `Address` within the `Order` class, followed by the `zipcode` property within the `Address` class, following manual traversal points.
-- If properties `Address` and `ZipCode` are found in the appropriate classes or their nested objects, the query will filter `Order` records based on the provided `Zipcode` object within the `Address` object.
+- It will then attempt to resolve the property named `address` within the `Order` class, followed by the `zipcode` property within the `MailingAddress` class, following manual traversal points.
+- If properties `address` and `zipCode` are found in the appropriate classes or their nested objects, the query will filter `Order` records based on the provided `zipcode` integer within the `MailingAddress` object.
```

Comment:
```suggestion
- If properties `address` and `zipcode` are found in the appropriate classes or their nested objects, the query will filter `Order` records based on the provided `zipcode` integer within the `MailingAddress` object.
```

Reply:
Good catch with the capitalization. I'll add your suggestion.